doc/deploying_gcs_on_bare_metal_machines.md

```diff
@@ -0,0 +1,196 @@
+### Deploying kubernetes + GCS on bare metal machines
+
+#### Cluster:
+
```

Comment:
I don't think we need step 1 to 7, its user choice, they can setup kubernetes as they want (either kubeadm, kubespray, hard way to setup kubernetes etc )

Reply:
May be we can mention that this is one way to install kubernetes and user could choose to install kubernetes in any other way. IMO if a user is using a clean setup then this guide will be a step to step guide to deploy GCS. Also, it provides descriptive explanantion on how to install kubernetes via the deploy-k8.yml script which is missing in gcs repo.